src/xpcc/architecture/platform/devices/stm32/stm32l476_486-j_m_q_r_v_z-c_e_g.xml

```diff
@@ -129,7 +129,7 @@
     <driver type="adc" name="stm32l4" instances="1,2,3"/>
     <driver type="can" name="stm32" instances="1"/>
     <driver type="clock" name="stm32"/>
-    <driver type="i2c" name="stm32" instances="1,2,3"/>
+    <driver type="i2c" name="stm32l4" instances="1,2,3"/>
```

Comment:
Is this driver also compatible with the other L4 devices? If so, can you also [change the DFG output](https://github.com/roboterclubaachen/xpcc/blob/develop/tools/device_file_generator/dfg/stm32/stm_writer.py#L77), so I don't forget when porting to modm?